src/Core/BoundedContexts/Headquarters/WB.Core.BoundedContexts.Headquarters/Storage/AmazonS3/InterviewS3FileStorage.cs

```diff
@@ -41,8 +41,8 @@ public virtual string GetPath(Guid interviewId, string filename = null)
     }
 
     protected abstract string GetInterviewDirectoryPath(Guid interviewId);
-    
-    protected abstract string ContentType { get; }
+
+    protected abstract string ContentType(string filename);
```

Comment:
What about ContentType -> GetContentType to be consistent with GetInterviewDirectoryPath?